### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by Fo4Ik-git**
- Add toast notification
- Add websocket reconnect 
- Add console commands for websocket connection

### CODE REVIEW

File: frontend/src/app/components/auth/auth.component.ts
Comment (Zephir0g) [inline frontend/src/app/components/auth/auth.component.ts:1]: clear structure but lacking error handling for the WebSocket connection and subscription processes

File: frontend/src/app/components/index/index.component.ts
Comment (Zephir0g) [inline frontend/src/app/components/index/index.component.ts:1]: ChatTemplate[]= [];             ?

File: frontend/src/app/config/Theme.config.ts
Comment (Zephir0g) [inline frontend/src/app/config/Theme.config.ts:1]: dark/light?

File: frontend/src/app/config/User.config.ts
Comment (Zephir0g) [inline frontend/src/app/config/User.config.ts:1]: g

File: frontend/src/app/services/toast.service.ts
Comment (Zephir0g) [inline frontend/src/app/services/toast.service.ts:1]: Toast? 
image

**Review by Zephir0g — APPROVED** [review]
хорошо!

File: frontend/src/app/components/auth/auth.component.ts
Comment (Fo4Ik-git) [reply frontend/src/app/components/auth/auth.component.ts:1]: Thats dev part only for testing

File: frontend/src/app/components/index/index.component.ts
Comment (Fo4Ik-git) [reply frontend/src/app/components/index/index.component.ts:1]: Array to messages from Server,  also for test

File: frontend/src/app/config/Theme.config.ts
Comment (Fo4Ik-git) [reply frontend/src/app/config/Theme.config.ts:1]: No matter, in future if add new theme we need send only it name and that will be work

File: frontend/src/app/config/User.config.ts
Comment (Fo4Ik-git) [reply frontend/src/app/config/User.config.ts:1]: What g?

File: frontend/src/app/services/toast.service.ts
Comment (Fo4Ik-git) [reply frontend/src/app/services/toast.service.ts:1]: )

**Pull request merged**